src/components/PlatformProvider/PlatformProvider.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { baselineComponent } from '../../testing/utils';
+import { ConfigProvider } from '../ConfigProvider/ConfigProvider';
+import {
+  ConfigProviderContextInterface,
+  useConfigProvider,
+} from '../ConfigProvider/ConfigProviderContext';
+import { PlatformProvider } from './PlatformProvider';
+
+describe('PlatformProvider', () => {
+  baselineComponent<any>(PlatformProvider, { forward: false });
+
+  describe('override config context', () => {
```

Comment:
```suggestion
  it('override config context', () => {
```

как-будто в `baselineComponent` нет смысла когда мы тестим провайдер

Reply:
При `forward: false` вообще нет смысла, да х)